src/script/sign.cpp

```diff
@@ -33,6 +33,23 @@ bool TransactionSignatureCreator::CreateSig(const SigningProvider& provider, std
     return true;
 }
 
+bool PSBTSignatureCreator::CreateSig(const SigningProvider& provider, std::vector<unsigned char>& vchSig, const CKeyID& address, const CScript& scriptCode, SigVersion sigversion) const
+{
+    CPubKey pubkey;
+    provider.GetPubKey(address, pubkey);
```

Comment:
In commit "Implement a signer that takes an unserialized PSBT and signs it":

Check result of `provider.GetPubKey()` and react if `false`?

Reply:
Done